ctf/check.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
@xudojnikomsk one function: `os.execute`

If you need to make it look nice, just redefine dofile with something that use it.
It wouldn't affect the instances created in `os.execute`.